docs/api.md

```diff
@@ -598,7 +598,8 @@ coll.create({title: "foo"}).then(_ => coll.sync())
 
 ### Local transformers
 
-In a near future, Kinto.js will provide transfomers aimed at providing facilities to encode and decode records when persisted locally.
+In a near future, Kinto.js will provide transfomers aimed at providing facilities to encode and decode records when persisted locally. If you are interested into
+making this happen, [don't hesitate to connect with us](https://kiwiirc.com/client/irc.freenode.net/?#kinto).
```

Comment:
Then I need to update the contributing page to have a "how to contact us" link :)